app/models/search.rb

```diff
@@ -0,0 +1,11 @@
+class Search < ActiveRecord::Base
+  belongs_to :user
+
+  def results
+    Development.periscope(query)
+  end
+
+  def unsaved?
```

Comment:
Basically, for a more readable public interface. Instead of asking `!resource.saved?`, we can ask `resource.unsaved?`.